### ISSUE TITLE
Decision tree with split training and test 4000 fewer in test set

### ISSUE THREAD

**Issue by arlyon** [arlyon/dmml#31]
> Make new training and testing sets, by moving 4000 of the instances from the original training set into the testing set. Then, repeat step 5.
